packages/daimo-common/src/landline.ts

```diff
@@ -0,0 +1,129 @@
+import { DaimoChain } from "@daimo/contract";
+import { Address, Hex, parseUnits } from "viem";
+
+import {
+  OffchainTransfer,
+  OpStatus,
+  TransferClog,
+  TransferSwapClog,
+} from "./op";
+import { dateStringToUnixSeconds, guessNumFromTimestamp } from "./time";
+
+export interface LandlineAccount {
+  daimoAddress: Address;
+  landlineAccountUuid: string;
+  bankName: string;
+  bankLogo: string | null;
+  accountName: string;
+  accountNumberLastFour: string;
+  bankCurrency: string;
+  liquidationAddress: Address;
+  liquidationChain: string;
+  liquidationCurrency: string;
+  createdAt: string;
+}
+
+export enum LandlineTransferStatus {
+  Processing = "processing",
+  Completed = "completed",
+  Failed = "failed",
+  Returned = "returned",
+}
+
+export enum LandlineTransferType {
+  Deposit = "deposit",
+  Withdrawal = "withdrawal",
+}
+
+export interface LandlineTransfer {
+  daimoAddress: Address;
```

Comment:
maybe worth having `LandlineTransfer` contain a `landlineAccount` vs copying in all the fields? either way works for me